pytest/tests/sanity/backward_compatible.py

```diff
@@ -65,13 +65,20 @@ def main():
     # TODO: send some transactions to test that runtime works the same.
     BLOCKS = 20
     TIMEOUT = 150
-    max_height = 0
+    max_height = -1
```

Comment:
why?

Reply:
For debugging purposes. Since first height is 0, starting at -1 it allows to print height 0 in the beginning.